config/proto/targets.proto

```diff
@@ -40,24 +40,15 @@ message TargetDefinition {
 
   TargetSystem target_system = 1;
   ConnectionType connection_type = 2;
-  ConnectionDetails connection_details = 3;
-  string target_url = 5;  // URL for connecting to the the API of the target storage system.
-  int64 total_space_allocated = 4;  // in MBs.
-}
 
-// REQUIRED, when using Ceph as the target system.
-// Comparable functionality is handled in the Go client library for GCS.
-message ConnectionDetails {
   // REQUIRED for Ceph S3 API
   // Port for connecting to Ceph S3 API
-  int32 port = 1;
+  int32 port = 3;
```

Comment:
I know this isn't used anywhere, but in general, changing the number of protos, but keeping the name is setting yourself up for the most painful debugging problem ever.

Reply:
oh, nm, you're consolidating existing protos.